### PR TITLE
feat(keyless-backup): settings item with setup and delete

### PR DESCRIPTION
**Pull request by satish-ravi**
### Description

Update settings item to match latest designs. Only includes setup and delete. In progress state is in a different issue

[Figma](https://www.figma.com/file/gGp6SVFWMbnSltkGzNzrpC/Wallet-Recovery?node-id=3604%3A30592&mode=dev)

### Test plan

Setup:

https://github.com/valora-inc/wallet/assets/5062591/a54719aa-019d-47dd-bf79-00dc784fcc32

Delete:

### Related issues

- Fixes ACT-765

### Backwards compatibility

Yes

### CODE REVIEW

File: src/keylessBackup/utils.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+// helper functions that default to false, but allows testing by mocking
+
+// TODO(ACT-771): remove when status taken from statsig
+export const getKeylessBackupGate = () => false
+
+// TODO(ACT-684, ACT-766, ACT-767): remove when status taken from redux
+export const isKeylessBackupCompleteUtil = () => false
```
Comment (jophish) [inline src/keylessBackup/utils.ts:7]: nit: I know this is going to eventually be deleted, but not sure I understand the need for the `Util` suffix

File: src/components/SettingsItem.tsx [hunk 2]
```diff
@@ -158,6 +158,40 @@ export function SettingsItemInput({
   )
 }
 
+type SettingsItemCtaProps = {
+  ctaText?: string | null
+  ctaColor?: colors
+  showChevron?: boolean
+} & BaseProps
+
+export function SettingsItemCta({
```
Comment (jophish) [inline src/components/SettingsItem.tsx:167]: I know this is a minor component, but could this use a test or two? Looks like the other components exported from this file have tests. I suppose this is already tested indirectly via the `Settings.test.tsx` file, however.

File: src/account/Settings.test.tsx [hunk 3]
```diff
@@ -302,6 +305,36 @@ describe('Account', () => {
...
+    const store = createMockStore()
+    const { getByTestId, getByText } = render(
+      <Provider store={store}>
+        <Settings {...getMockStackScreenProps(Screens.Settings)} />
+      </Provider>
+    )
+    expect(getByTestId('KeylessBackup')).toBeTruthy()
+    expect(getByText('delete')).toBeTruthy()
+    fireEvent.press(getByTestId('KeylessBackup'))
+    expect(navigate).not.toHaveBeenCalled()
```
Comment (jophish) [inline src/account/Settings.test.tsx:335]: Can we ensure that analytics get fired here?

File: src/account/Settings.test.tsx [hunk 4]
```diff
@@ -302,6 +305,36 @@ describe('Account', () => {
...
+    const store = createMockStore()
+    const { getByTestId, getByText } = render(
+      <Provider store={store}>
+        <Settings {...getMockStackScreenProps(Screens.Settings)} />
+      </Provider>
+    )
+    expect(getByTestId('KeylessBackup')).toBeTruthy()
+    expect(getByText('setup')).toBeTruthy()
+    fireEvent.press(getByTestId('KeylessBackup'))
+    expect(navigate).toHaveBeenCalledWith(Screens.WalletSecurityPrimer)
```
Comment (jophish) [inline src/account/Settings.test.tsx:320]: same here with analytics

File: src/keylessBackup/utils.ts (hunk 1 above)
Comment (satish-ravi) [reply src/keylessBackup/utils.ts]: haha I was just been lazy with naming, updated

File: src/components/SettingsItem.tsx (hunk 2 above)
Comment (satish-ravi) [reply src/components/SettingsItem.tsx:167]: added

File: src/account/Settings.test.tsx (hunk 3 above)
Comment (satish-ravi) [reply src/account/Settings.test.tsx:339]: done

File: src/account/Settings.test.tsx (hunk 4 above)
Comment (satish-ravi) [reply src/account/Settings.test.tsx:320]: done

**Review by jh2oman — APPROVED** [review]

**Review by jophish — APPROVED** [review]

**Pull request merged**